tools/modelgen/generators/templates/register.tmpl

```diff
@@ -0,0 +1,5 @@
+{{ range .Objects }}
+func Register{{ initialCap .Name }}Callbacks(handler {{ initialCap .Name }}Callbacks) {
+  objCallbackHandler.{{ initialCap .Name }}Cb = handler
```

Comment:
If we used a struct for these instead of interfaces, we could just `nil` check and handle at the top level with a default handler, no point in having extra methods for it.